<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestaoColaboradores</class>
 <widget class="QDialog" name="fm_gestaoColaboradores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_novocaborador">
      <attribute name="title">
       <string>Novo Colaborador</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>701</width>
         <height>271</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="lbael2">
           <property name="text">
            <string>Nome:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txt_novo_colab"/>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Nome de usuario:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txt_username_novocolab"/>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Senha:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txt_senha_novo_colab">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Telefone:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txt_fone_novocolab"/>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tipo de acesso:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QComboBox" name="cb_novo_acesso"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter_7">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>290</y>
         <width>511</width>
         <height>28</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="btn_gravar_novocolab">
        <property name="text">
         <string>Gravar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_can_novo_novocolab">
        <property name="text">
         <string>Novo</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_cancalar_novocolab">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Gest√£o de Colaboradores</string>
      </attribute>
      <widget class="QPushButton" name="btn_flitro_colab">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>80</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Filtrar</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>106</width>
         <height>50</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rb_idcolab">
          <property name="text">
           <string>Id Cobal.</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_nome_colab">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>20</y>
         <width>137</width>
         <height>52</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Filtro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_filtro_colab"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>411</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QSplitter" name="splitter_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nome</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txt_nome_colab"/>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txt_user_cobal"/>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Senha</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txt_senha_colab">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Telefone</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txt_colab_tel"/>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Acesso</string>
           </property>
          </widget>
          <widget class="QComboBox" name="cb_acesso_colabt"/>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QPushButton" name="btn_editar_colab">
           <property name="text">
            <string>Editar</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_excluir_colab">
           <property name="text">
            <string>Excluir</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_vendas_colab">
           <property name="text">
            <string>Ver Vendas</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableWidget" name="lv_colab">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>140</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
